deploy/helm/charts/README.md

```diff
@@ -0,0 +1,143 @@
+| csiNode.driverRegistrar.image.registry | string | `"k8s.gcr.io/"` | CSI Node driver registrar image registry |
+| csiNode.driverRegistrar.image.repository | string | `"k8scsi/csi-node-driver-registrar"` | CSI Node driver registrar image repository |
+| csiNode.driverRegistrar.image.tag | string | `"v2.0.1"` |  CSI Node driver registrar image tag|
+| csiNode.driverRegistrar.name | string | `"csi-node-driver-registrar"` | CSI Node driver registrar container name |
+| csiNode.kubeletDir | string | `"/var/lib/kubelet/"` | Kubelet root dir |
+| csiNode.labels | object | `{}` | CSI Node pod labels |
+| csiNode.nodeSelector | object | `{}` |   CSI Node pod nodeSelector |
+| csiNode.podAnnotations | object | `{}` | CSI Node pod annotations |
+| csiNode.resources | object | `{}` | CSI Node pod resources |
+| csiNode.securityContext | object | `{}` | CSI Node pod security context |
+| csiNode.tolerations | list | `[]` | CSI Node pod tolerations |
+| csiNode.updateStrategy.type | string | `"RollingUpdate"` | CSI Node daemonset update strategy |
+| csiNode.livenessprobe.image.pullPolicy | string | `"IfNotPresent"` | CSI livenessprobe image pull policy |
+| csiNode.livenessprobe.image.registry | string | `"k8s.gcr.io/"` |  CSI livenessprobe image registry |
+| csiNode.livenessprobe.image.repository | string | `"k8scsi/livenessprobe"` |  CSI livenessprobe image repo |
+| csiNode.livenessprobe.image.tag | string | `"v2.2.0"` | CSI livenessprobe image tag |
+| csiNode.livenessprobe.name | string | `"liveness-probe"` |  CSI livenessprobe container name|
+| jivaOperator.annotations | object | `{}` | Jiva operator annotations |
+| jivaOperator.componentName | string | `"jiva-operator"` | Jiva operator component name |
+| jivaOperator.image.pullPolicy | string | `"IfNotPresent"` | Jiva operator image pull policy |
+| jivaOperator.image.registry | string | `nil` | Jiva operator image registry |
+| jivaOperator.image.repository | string | `"openebs/jiva-operator"` | Jiva operator image repository |
+| jivaOperator.image.tag | string | `"2.6.0"` |  Jiva operator image tag |
+| jivaOperator.nodeSelector | object | `{}` |  Jiva operator pod nodeSelector|
+| jivaOperator.podAnnotations | object | `{}` | Jiva operator pod annotations |
+| jivaOperator.resources | object | `{}` | Jiva operator pod resources |
+| jivaOperator.securityContext | object | `{}` | Jiva operator security context |
+| jivaOperator.tolerations | list | `[]` | Jiva operator pod tolerations |
+| jivaCSIPlugin.image.pullPolicy | string | `"IfNotPresent"` | Jiva CSI driver image pull policy |
+| jivaCSIPlugin.image.registry | string | `nil` | Jiva CSI driver image registry |
+| jivaCSIPlugin.image.repository | string | `"openebs/jiva-csi"` |  Jiva CSI driver image repository |
+| jivaCSIPlugin.image.tag | string | `"2.6.0"` | Jiva CSI driver image tag |
+| jivaCSIPlugin.name | string | `"jiva-csi-plugin"` | Jiva CSI driver container name |
```

Comment:
nit:
`jivaCSIPlugin.remount` missing

Reply:
that's was intentionally left , don't want to list that here.
But added now